notebooks/lesson10_clusters.ipynb

```diff
@@ -0,0 +1,842 @@
+{
```

Comment:
<p>I would move the imports and helper functions to be above part 1 and make them level 2 headings <code>## Imports</code>  etc</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>